spec/lib/metasploit/framework/login_scanner/http_spec.rb

```diff
@@ -30,4 +30,16 @@
     end
   end
 
+  describe '#set_http_trace_proc' do
+    it 'configures http tracing' do
+      expect(subject.set_http_trace_proc(true)).to be_kind_of(Proc)
+    end
+  end
+
+  describe '#set_http_trace_proc' do
+    it 'does not log http tracing' do
```

Comment:
These descriptions are too vauge. Configures HTTP tracing when what occurs? Does not log HTTP tracing when what occurs? Need to be more specific, otherwise its going to be harder for readers of RSpec output to determine what exactly went wrong.